src/CommonItilObject_Item.php

```diff
@@ -390,7 +341,7 @@ public static function showForObject($obj, $options = [])
                     [static::$items_id_1 => $instID]
                 );
             }
-            $used = static::getUsedItems($instID);
+            $used = static::getUsedItems($instID, Ticket::class);
```

Comment:
Is this supposed to be statically set for tickets instead of `static::$itemtype_1`?

Reply:
done